pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -258,9 +270,10 @@ message CommandAck {
 		DecompressionError = 1;
 		ChecksumMismatch = 2;
 		BatchDeSerializeError = 3;
+		DecryptionError = 4;
 	}
 
-	optional ValidationError validation_error = 4;
+	optional ValidationError validation_error = 5;
```

Comment:
We don't have to change field value here?

Reply:
Isn't it supposed to be set to a value outside the ValidationError enum?

Reply:
AFAIK - It is the field number, not the value - so needs to remain 4.

Reply:
Why would we send BatchDeSerializeError if consumer received invalid message?